sjb/config/test_cases/test_branch_online_console_extensions.yml

```diff
@@ -0,0 +1,13 @@
+---
+parent: 'common/test_cases/origin.yml'
+overrides:
+  sync_repos:
+    - name: "online-console-extensions"
+  junit_analysis: False
+extensions:
+  actions:
+    - type: "script"
+      title: "online-console-extensions generated template check"
+      repository: "online-console-extensions"
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/scripts JUNIT_REPORT='true' ./hack/check-generated-templates.sh
```

Comment:
You probably don't need any of these env vars

Reply:
you're probably right :) thanks